### DISCUSSION TITLE
Any possibilities to add vim like features?

### DISCUSSION THREAD

**Discussion by Izaird** [d99kris/nchat#124] [Q&A]
I love the idea of this app, I think a great addittion will be to have vim movement and also vim mode to edit messages

**Comment by d99kris** [comment]
Hi @Izaird - thanks for the feedback. Unfortunately there are no plans to make the UI modal like vim. The default keybindings UI is inspired by those of emacs / nano / bash.

You may use your favourite `$EDITOR` to compose messages in nchat by pressing `alt-e` (`opt-e` on mac).
